<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEditName">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditYear">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditID">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Year</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="allDataViewButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>101</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Вывести всё</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filterData2Button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>84</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Год &lt; 2001</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filterDataButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>84</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Год &gt; 2000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRem">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>111</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить запись</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAdd">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>111</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить запись</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
